sos/plugins/pmem.py

```diff
@@ -0,0 +1,83 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin, UbuntuPlugin, DebianPlugin
+
+
+class Pmem(Plugin, RedHatPlugin, UbuntuPlugin, DebianPlugin):
+    '''Persistent Memory Device Plugin
+
+    This plugin collects data from Persistent Memory Modules,
+    commonly referred to as NVDIMM's or Storage Class Memory (SCM).
+    '''
+
+    plugin_name = 'pmem'
+    profiles = ('storage', 'hardware', 'memory')
+    # Utilities can be installed by package or self compiled
+    packages = ('ndctl', 'ipmctl', 'ixpdimm-cli')
+    files = ('/usr/local/bin/ndctl',
```

Comment:
Instead of `files` use `commands` for this.

E.G.:
```python
commands = ('ndctl', 'ipmctl', 'ixpdimm-cli')
```

Reply:
I've implemented this suggestion.  Thanks!